dspace-server-webapp/src/test/java/org/dspace/app/rest/SystemWideAlertRestRepositoryIT.java

```diff
@@ -0,0 +1,299 @@
+        SystemWideAlert systemWideAlert1 = systemWideAlertService.create(context, "Test alert 1",
+                                                                         AllowSessionsEnum.ALLOW_CURRENT_SESSIONS_ONLY,
+                                                                         countdownDate,
+                                                                         true);
+        SystemWideAlert systemWideAlert2 = systemWideAlertService.create(context, "Test alert 2",
+                                                                         AllowSessionsEnum.ALLOW_ADMIN_SESSIONS_ONLY,
+                                                                         null,
+                                                                         false);
+        DateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
+        getClient().perform(get("/api/system/systemwidealerts/"))
+                   .andExpect(status().isOk())
+                   .andExpect(jsonPath("$._embedded.systemwidealerts", containsInAnyOrder(
+                           allOf(
+                                   hasJsonPath("$.alertId", is(systemWideAlert1.getID())),
+                                   hasJsonPath("$.message", is(systemWideAlert1.getMessage())),
+                                   hasJsonPath("$.allowSessions", is(systemWideAlert1.getAllowSessions())),
+                                   hasJsonPath("$.countdownTo",
+                                               startsWith(sdf.format(systemWideAlert1.getCountdownTo()))),
+                                   hasJsonPath("$.active", is(systemWideAlert1.isActive()))
+                           ),
+                           allOf(
+                                   hasJsonPath("$.alertId", is(systemWideAlert2.getID())),
+                                   hasJsonPath("$.message", is(systemWideAlert2.getMessage())),
+                                   hasJsonPath("$.allowSessions", is(systemWideAlert2.getAllowSessions())),
+                                   hasJsonPath("$.countdownTo", is(systemWideAlert2.getCountdownTo())),
+                                   hasJsonPath("$.active", is(systemWideAlert2.isActive()))
+                           )
+                   )));
+
+
+    }
+
+    @Test
+    public void findOneTest() throws Exception {
+
+        Date countdownDate = new Date();
+        SystemWideAlert systemWideAlert1 = systemWideAlertService.create(context, "Test alert 1",
+                                                                         AllowSessionsEnum.ALLOW_CURRENT_SESSIONS_ONLY,
+                                                                         countdownDate,
+                                                                         true);
+        SystemWideAlert systemWideAlert2 = systemWideAlertService.create(context, "Test alert 2",
+                                                                         AllowSessionsEnum.ALLOW_ADMIN_SESSIONS_ONLY,
```

Comment:
Again, in this test we are creating 2 alerts, which is not supported by our REST endpoint.  These ITs are meant to test the REST endpoint, so it's odd to me that they start by creating a scenario which is not supported by the endpoint.